pkg/identity/numericidentity.go

```diff
@@ -75,6 +75,10 @@ const (
 	// received any labels yet.
 	ReservedIdentityInit
 
+	// ReservedIdentityNonLocalNode is the identity given to worker nodes
```

Comment:
Can you be a little more specific here?

I assume "to worker nodes in any connected cluster" should be "to worker nodes, running a cilium-agent, in any connected cluster"

Reply:
Not really, we also identify nodes where Cilium is not running on.

Reply:
I've clarified the comment a bit